ballerina-tests/tests/01_advanced_type_service.bal

```diff
@@ -27,66 +27,47 @@ listener grpc:Listener ep = new (9091, {
 }
 service "HelloWorld" on ep {
 
-    isolated remote function testInputNestedStruct(HelloWorldStringCaller caller, Person req) {
-        io:println("name: " + req.name);
+    isolated remote function testInputNestedStruct(HelloWorldStringCaller caller, Person req) returns grpc:Error? {
+        log:printInfo("name: " + req.name);
         string message = "Submitted name: " + req.name;
-        io:println("Response message " + message);
-        grpc:Error? err = caller->sendString(message);
-        if err is grpc:Error {
-            io:println("Error from Connector: " + err.message());
-        }
-        checkpanic caller->complete();
+        check caller->sendString(message);
```

Comment:
Here error can come, when the message can't deliver to the client due to network failure etc. is it correct to send the error back.

Reply:
Do you mean something like when a network failure occurs, we should handle it at the server-side instead of returning it?.

Reply:
yes exactly.

Reply:
I thought since this is a test, it would be okay to test such network failures.